Cargo.toml

```diff
@@ -37,6 +37,8 @@ serde = { version = "1.0.114", features = ["derive"] }
 dirs = "3.0.1"
 itertools = "0.9.0"
 ordered-float = "2.0.0"
+cached-path = "0.4.0-rc1"
```

Comment:
could you please ensure a release is added as a dependency for the final version of this PR?